test/spec/__snapshots__/mfa-challenge.js.snap

```diff
@@ -0,0 +1,82 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
is this the first test using `jest`? can we have jasmine and jest in this repo with no prob? or should we stick to jasmine for now?

Reply:
I think all the tests are moved to jest already. I guess this is the first one using snapshots. 

The jasmine based tests are under https://github.com/okta/okta-auth-js/tree/master/test/karma

https://github.com/okta/okta-auth-js/pull/252/files#diff-9dd9cfed8da8bb5fe5cd6c5cddb0785fR1